### PR TITLE
Make several minor copy fixes and add the About page

### PR DESCRIPTION
**Pull request by QuincyLarson**

### CODE REVIEW

File: server/views/resources/links.jade [hunk 1]
```diff
@@ -2,57 +2,145 @@ extends ../layout
...
+                                    a(href="/stories") Stories from campers who've become professional developers
+                        .spacer
+                        h2 Which camper to contact for:
+                        table.table.table-stripe
+                            tr
+                                td Support (team@freecodecamp.com)
+                                td
+                                    a(href='https://gitter.im/QuincyLarson' target='_blank') @QuincyLarson
+                            tr
+                                td Facebook page
```
Comment (SaintPeter) [inline server/views/resources/links.jade]: I think all of these should be in Title case:
Facebook Page
Twitter Feed
etc.

Some are in title case, some are not.

File: server/views/resources/terms-and-privacy.jade [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+                h3
+                    a#Personally_Identifiable_Information_2
+                    | Personally Identifiable Information
+                p
+                    | Free Code Camp protects the identity of visitors to
+                    a(href='http://FreeCodeCamp.com') FreeCodeCamp.com
+                    |  by limiting the collection of personally identifiable information.
+                p
+                    | Free Code Camp does not knowingly collect or solicit personally identifiable information from or about children under 13, except as permitted by law. If we discover we have received any information from a child under 13 in violation of this policy, we will delete that information immediately. If you believe Free Code Camp has any information from or about anyone under 13, please email us at
+                    a(href='mailto:team@freecodecamp.com') team@freecodecamp.com
```
Comment (SaintPeter) [inline server/views/resources/terms-and-privacy.jade]: Missing a space between "at" and "team@freecodecamp.com" when rendered.

File: server/views/resources/terms-and-privacy.jade [hunk 3]
```diff
@@ -0,0 +1,142 @@
...
+                    a#Anonymous_Information_15
+                    | Anonymous Information
+                p
+                    | Anonymous aggregated data may be provided to other organizations we associate with for statistical purposes. For example, we may report to an organization that a certain percentage of our site’s visitors are adults between the ages of 25 and 35.
+                h3
+                    a#Cookies_and_BeaconsUse_by_Free_Code_Camp_Opting_Out_18
+                    | Cookies and Beacons—Use by Free Code Camp; Opting Out
+                p
+                    | We use cookies and software logs to monitor the use of
+                    a(href='http://FreeCodeCamp.com') FreeCodeCamp.com
```
Comment (SaintPeter) [inline server/views/resources/terms-and-privacy.jade]: Missing space before the link

File: server/views/resources/terms-and-privacy.jade [hunk 4]
```diff
@@ -0,0 +1,142 @@
...
+                    |  and to gather non-personal information about visitors to the site. Cookies are small files that Free Code Camp transfers to the hard drives of visitors for record-keeping purposes. These monitoring systems allow us to track general information about our visitors, such as the type of browsers (for example, Firefox or Internet Explorer), the operating systems (for instance, Windows or Macintosh), or the Internet providers (for instance, Comcast) they use. This information is used for statistical and market research purposes to tailor content to usage patterns and to provide services requested by our customers. To delete these cookies, please see your browser’s privacy settings.
+                p
+                    | A beacon is an electronic file object (typically a transparent image) placed in the code of a Web page. We use third party beacons to monitor the traffic patterns of visitors from one Free Code
+                    a(href='http://Camp.com') Camp.com
```
Comment (SaintPeter) [inline server/views/resources/terms-and-privacy.jade]: Only the last word is a link, and it's linking to "Camp.com"
image

File: server/views/resources/terms-and-privacy.jade [hunk 5]
```diff
@@ -0,0 +1,142 @@
...
+                    a#Surveys_28
+                    | Surveys
+                p
+                    | We may occasionally conduct on-line surveys. All surveys are voluntary and you may decline to participate.
+                h3
+                    a#Copyright_31
+                    | Copyright
+                p
+                    | All of the content on
+                    a(href='http://FreeCodeCamp.com') FreeCodeCamp.com
```
Comment (SaintPeter) [inline server/views/resources/terms-and-privacy.jade]: Missing a space after "on"

File: server/views/resources/terms-and-privacy.jade [hunk 6]
```diff
@@ -0,0 +1,142 @@
...
+                    |  is copyrighted by Free Code Camp. If you’d like to redistribute it beyond simply sharing it through social media, please contact us at
+                    a(href='mailto:team@freecodecamp.com') team@freecodecamp.com
```
Comment (SaintPeter) [inline server/views/resources/terms-and-privacy.jade]: Missing a space after "at"

File: server/views/resources/terms-and-privacy.jade [hunk 7]
```diff
@@ -0,0 +1,142 @@
...
+                    | .
+                h3
+                    a#Contacting_Us_34
+                    | Contacting Us
+                p
+                    | If you have questions about Free Code Camp, or to correct, update, or remove personally identifiable information, please email us at
+                    a(href='mailto:team@freecodecamp.com') team@freecodecamp.com
```
Comment (SaintPeter) [inline server/views/resources/terms-and-privacy.jade]: Missing a space before the url.

File: server/views/resources/terms-and-privacy.jade [hunk 8]
```diff
@@ -0,0 +1,142 @@
...
+                    | .
+                h3
+                    a#Links_to_Other_Web_sites_37
+                    | Links to Other Web sites
+                p
+                    | Free Code Camp’s sites each contain links to other Web sites. Free Code Camp is not responsible for the privacy practices or content of these third-party Web sites. We urge all
+                    a(href='http://FreeCodeCamp.com') FreeCodeCamp.com
```
Comment (SaintPeter) [inline server/views/resources/terms-and-privacy.jade]: Missing a space after "all", before the link.

File: server/views/resources/terms-and-privacy.jade [hunk 9]
```diff
@@ -0,0 +1,142 @@
...
+                h3
+                    a#Your_California_Privacy_Rights_46
+                    | Your California Privacy Rights
+                p
+                    | If you are a California resident, you are entitled to prevent sharing of your personal information with third parties for their own marketing purposes through a cost-free means. If you send a request to the address above, Free Code Camp will provide you with a California Customer Choice Notice that you may use to opt-out of such information sharing. To receive this notice, submit a written request to <a
+                    a(href='mailto:href=%22mailto:team@freecodecamp.com') href="mailto:team@freecodecamp.com
+                    | " target="_blank">team@freecodecamp.com</a>, specifying that you seek your “California Customer Choice Notice.” Please allow at least thirty (30) days for a response.
```
Comment (SaintPeter) [inline server/views/resources/terms-and-privacy.jade]: Broken link.  Probably needs to be converted to Jade?

File: server/views/resources/terms-and-privacy.jade [hunk 10]
```diff
@@ -0,0 +1,142 @@
...
+                h3
+                    a#Acceptance_of_Privacy_Policy_Terms_and_Conditions_49
+                    | Acceptance of Privacy Policy Terms and Conditions
+                p
+                    | By using this site, you signify your agreement to the terms and conditions of this
+                    a(href='http://FreeCodeCamp.com') FreeCodeCamp.com
```
Comment (SaintPeter) [inline server/views/resources/terms-and-privacy.jade]: Missing Space

File: server/views/resources/terms-and-privacy.jade [hunk 11]
```diff
@@ -0,0 +1,142 @@
...
+                    |  Privacy Policy. If you do not agree to these terms, please do not use this site. We reserve the right, at our sole discretion, to change, modify, add, or remove portions of this policy at any time. All amended terms automatically take effect 30 days after they are initially posted on the site. Please check this page periodically for any modifications. Your continued use of
+                    a(href='http://FreeCodeCamp.com') FreeCodeCamp.com
```
Comment (SaintPeter) [inline server/views/resources/terms-and-privacy.jade]: Missing space

File: server/views/resources/terms-and-privacy.jade [hunk 12]
```diff
@@ -0,0 +1,142 @@
...
+                    |  following the posting of any changes to these terms shall mean that you have accepted those changes.
+                p
+                    | If you have any questions or concerns, please send an email to
```
Comment (SaintPeter) [inline server/views/resources/terms-and-privacy.jade]: Missing space

File: server/views/resources/terms-and-privacy.jade [hunk 13]
```diff
@@ -0,0 +1,142 @@
...
+                p
+                    | We do not tolerate harassment of campers in any form, anywhere on Free Code Camp’s online media (Gitter, Twitch, Facebook, etc.) or during pair programming. Harassment includes sexual language and imagery, deliberate intimidation, stalking, unwelcome sexual attention, libel, and any malicious hacking or social engineering.
+                p
+                    | If a camper engages in harassing behavior, our team will take any action we deem appropriate, up to and including banning them from Free Code Camp.
+                p
+                    | No bots are allowed in any of the Official Chat Rooms without prior explicit permission from the FCC Core Team.
+                p
+                    | We want everyone to feel safe and respected. If you are being harassed or notice that someone else is being harassed, say something! Message @quincylarson, @berkeleytrue, @brianamarie and @codenonprofit in Gitter (preferably with a screen shot of the offending language) so we can take fast action.
+                p
+                    | If you have questions about this code of conduct, email us at
```
Comment (SaintPeter) [inline server/views/resources/terms-and-privacy.jade]: Missing Space

**Comment by SaintPeter** [comment]
Aside from the above, it functions properly and looks good.

**Comment by BerkeleyTrue** [comment]
merged

**Pull request closed without merging**